docs/ingesting_data/Ingesting_data_with_panpipes.md

```diff
@@ -5,10 +5,19 @@ Panpipes is a single cell multimodal analysis pipeline with a lot of functionali
 Arguably, the most important part of a pipeline is the ingestion of the data into a format that allows efficient storage and agile processing. We believe that `AnnData` and `MuData` offer all those advantages and that's why we built `panpipes` with these data structure at its core. 
 Please check the [`scverse`](https://scverse.org/) webpage for more information on these formats!
 
+When running panpipes, you always need to start with the data ingestion. The ingest workflow will do the following:
+- create summary plots of 10x metrics
+- calculate scrublet scores (doublet detection)
+- scanpy Quality Control (QC)
+- create summary QC plots
+
+The ingest pipeline does not perform any filtering of cells or genes. Filtering occurs as the first step in the [preprocess workflow](https://panpipes-tutorials.readthedocs.io/en/latest/filtering_data/filtering_data_with_panpipes.html), which should be executed after the ingestion workflow.
+
 We provide examples of how to ingest single cell data from a 10X directory (see the [multiome tutorial](https://panpipes-tutorials.readthedocs.io/en/latest/ingesting_multiome/ingesting_mome.html) or the [citeseq tutorial]()) or directly from existing anndata objects, but we offer the possibility load any tabular format and assay-specific data types into a MuData object (check the [Supported Input Filetypes](https://panpipes-pipelines.readthedocs.io/en/latest/usage/setup_for_qc_mm.html#supported-input-filetypes:~:text=per_barcode_metrics_file-,Supported%20input%20filetypes,-%EF%83%81) section and our [Ingesting Spatial Transcriptomics Tutorial](https://panpipes-tutorials.readthedocs.io/en/latest/ingesting_spatial_data/Ingesting_spatialdata_with_panpipes.html))
```

Comment:
link for the citeseq tutorial 
https://panpipes-tutorials.readthedocs.io/en/latest/ingesting_multimodal_data/ingesting_multimodal_data.html